indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -23,6 +23,51 @@ $(document).ready(function() {
         });
     });
 
+    var gradientLayer = $('.participant-list-wrapper > .gradient-layer');
+    var heightControl = $('<div>').addClass('height-control');
+    gradientLayer.append(heightControl);
+    var threeRowsHeight = heightControl.height();
+    $('.participant-list-wrapper').toggleClass('collapsible collapsed',
+        $('.participant-list').height() > threeRowsHeight);
+    var initialHeight = $('.participant-list-wrapper').height();
+    heightControl.remove();
+
+    $('.participant-list-wrapper > .trigger, .participant-list-wrapper > .gradient-layer').on('click', function() {
+        var toggler = $('.participant-list-wrapper > .trigger');
+        var participantList = toggler.siblings('.participant-list');
+        var wrapper = participantList.parent();
+        if (wrapper.hasClass('collapsed')) {
+            var newHeight = participantList.height();
+            participantList.height(initialHeight);
+            wrapper.find('.gradient-layer').fadeOut();
+            wrapper.removeClass('collapsed');
+            wrapper.animate({
+                height: newHeight
+            }, {
+                duration: 'fast',
+                start: function() {
+                    toggler.toggleClass('icon-expand icon-collapse');
+                },
+                complete: function() {
+                    participantList.height(newHeight);
+                }
+            });
+        } else {
+            wrapper.find('.gradient-layer').fadeIn();
+            wrapper.animate({
+                height: initialHeight,
+            }, {
+                duration: 'fast',
+                start: function() {
+                    toggler.toggleClass('icon-expand icon-collapse');
```

Comment:
Why call `addClass` and `removeClass` when I can trigger both of them with the `toggleClass`?